packages/frontend/src/pages/MetricsCatalog.tsx

```diff
@@ -16,59 +14,14 @@ const MetricsCatalog: FC<MetricsCatalogProps> = ({
 }) => {
     return (
         <Provider store={store}>
-            <MantineProvider
-                theme={{
-                    // TODO: Introduce Inter as a font in the theme globally
-                    ...getMantineThemeOverride(),
-                    fontFamily: `Inter, ${
-                        getMantineThemeOverride().fontFamily
-                    }`,
-                    components: {
-                        ...getMantineThemeOverride().components,
-                        Tooltip: {
-                            defaultProps: {
-                                openDelay: 200,
-                                withinPortal: true,
-                                withArrow: true,
-                                multiline: true,
-                                maw: 250,
-                                fz: 'xs',
-                            },
-                        },
-                        Popover: {
-                            defaultProps: {
-                                withinPortal: true,
-                                radius: 'md',
-                                shadow: 'sm',
-                            },
-                        },
-                        Paper: {
-                            defaultProps: {
-                                radius: 'md',
-                                shadow: 'subtle',
-                                withBorder: true,
-                                sx: (theme: MantineTheme) => ({
-                                    '&[data-with-border]': {
-                                        border: `1px solid ${theme.colors.gray[2]}`,
-                                    },
-                                }),
-                            },
-                        },
-                    },
-                }}
+            <Page
+                withCenteredRoot
+                withCenteredContent
+                withXLargePaddedContent
+                withLargeContent
             >
```

Comment:
double check

Reply:
removed the default props from Mantine Theme and moved them to all components in metrics catalog instead